<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Otter::PasswordsContentsWidget</class>
 <widget class="QWidget" name="Otter::PasswordsContentsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,0">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="Otter::LineEditWidget" name="filterLineEditWidget">
     <property name="placeholderText">
      <string>Searchâ€¦</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Otter::ItemViewWidget" name="passwordsViewWidget">
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <attribute name="headerDefaultSectionSize">
      <number>250</number>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="detailsWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <widget class="QWidget" name="formWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="domainLabel">
           <property name="text">
            <string>Domain:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="Otter::TextLabelWidget" name="domainLabelWidget" native="true"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="actionsLayout" stretch="0,0,1">
        <item>
         <widget class="QPushButton" name="showPasswordsButton">
          <property name="text">
           <string>Show Passwords</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deleteButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Otter::ItemViewWidget</class>
   <extends>QTreeView</extends>
   <header>src/ui/ItemViewWidget.h</header>
  </customwidget>
  <customwidget>
   <class>Otter::LineEditWidget</class>
   <extends>QLineEdit</extends>
   <header>src/ui/LineEditWidget.h</header>
  </customwidget>
  <customwidget>
   <class>Otter::TextLabelWidget</class>
   <extends>QWidget</extends>
   <header>src/ui/TextLabelWidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>filterLineEditWidget</tabstop>
  <tabstop>passwordsViewWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
